uaclient/contract.py

```diff
@@ -16,21 +16,31 @@
 from uaclient.defaults import ATTACH_FAIL_DATE_FORMAT
 from uaclient.entitlements.entitlement_status import UserFacingStatus
 
+# We are now describing each endpoint we use by the
+# type of HTTP request we are performing together with
+# endpoint name defined in the Contract backend. We must do
+# that to easily identify which endpoints are being used
+# by the Pro client
 API_V1_ADD_CONTRACT_MACHINE = "/v1/context/machines/token"
-API_V1_TMPL_CONTEXT_MACHINE_TOKEN_RESOURCE = (
+API_V1_GET_CONTRACT_MACHINE = (
     "/v1/contracts/{contract}/context/machines/{machine}"
 )
-API_V1_RESOURCES = "/v1/resources"
-API_V1_TMPL_RESOURCE_MACHINE_ACCESS = (
+API_V1_UPDATE_CONTRACT_MACHINE = (
+    "/v1/contracts/{contract}/context/machines/{machine}"
+)
+API_V1_GET_AVAILABLE_RESOURCES = "/v1/resources"
```

Comment:
Nitpick: it should have been named `GetAvailableResources`, but the official name on our side is just `AvailableResources`.

So I'd suggest renaming this to `API_V1_AVAILABLE_RESOURCES` to keep it fully aligned with the backend.